.changeset/sour-comics-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+"@turnkey/crypto": major
+"@turnkey/encoding": minor
+---
+
+@turnkey/crypto added decrypting for export bundles, changed decryptBundle to decryptEmailBundle (BREAKING CHANGE), added hexToAscii function to @turnkey/encoding for conversion to mnemonic
```

Comment:
can wait for the ensuing `pnpm run changeset version` PR (the release PR), but thinking we could do just a little bit of rearranging. maybe something like

```
---
"@turnkey/crypto": major
"@turnkey/encoding": minor
---

@turnkey/crypto
- [BREAKING CHANGE] renamed `decryptBundle` to `decryptEmailBundle` (for decrypting email auth credential bundles) in order to distinguish from the new `decryptExportBundle` (for decrypting bundles containing wallet mnemonics or private key material)

@turnkey/encoding
- added hexToAscii function, useful for converting a raw hex string to a (wallet) mnemonic
```